sgkit/utils.py

```diff
@@ -64,3 +64,55 @@ def encode_array(x: ArrayLike) -> Tuple[ArrayLike, List[Any]]:
     rank = np.empty_like(index)
     rank[index] = np.arange(len(index))
     return rank[inverse], names[index]
+
+
+def split_array_chunks(n: int, blocks: int) -> List[int]:
```

Comment:
Would it make more sense to return a `Tuple`?

Reply:
It could be Tuple[int, ...], why though?

Reply:
Immutability, when both make sense, immutable data structure in my experience, tend to be less error prone and easier to work with.

Reply:
https://github.com/pystatgen/sgkit/pull/69/commits/bda112f96279b909983e3b9c7e47234d6ea9a973